fix: clarify free regens (#776)
elevenlabs/elevenlabs-docs: fern/docs/pages/capabilities/text-to-speech.mdx

    available with paid plans. With a paid subscription, you may use generated audio for commercial
    purposes and monetize the outputs if you own the IP rights to the input content.
  </Accordion>
  <Accordion title="What qualifies as a free regeneration?">
    A free regeneration allows you to regenerate the same text to speech content without additional cost, subject to these conditions:

    - You can regenerate each piece of content up to 2 times for free
    - The content must be exactly the same as the previous generation. Any changes to the text, voice settings, or other parameters will require a new, paid generation

>     Free regenerations are useful in case there is a slight distortion in the audio output. According to ElevenLab's internal benchmarks, regenerations will solve roughly half of issues with quality, with remaining issues usually due to poor training data.

Reviewer: ```suggestion
    Free regenerations are useful in case there is a slight distortion in the audio output. According to ElevenLabs' internal benchmarks, regenerations will solve roughly half of issues with quality, with remaining issues usually due to poor training data.
```

    available with paid plans. With a paid subscription, you may use generated audio for commercial
    purposes and monetize the outputs if you own the IP rights to the input content.
  </Accordion>
  <Accordion title="What qualifies as a free regeneration?">

>     A free regeneration allows you to regenerate the same text-to-speech content without additional cost, subject to these conditions:

Reviewer: `text-to-speech` is apparently not good SEO, so we're moving away from it.

```suggestion
    A free regeneration allows you to regenerate the same text to speech content without additional cost, subject to these conditions:
```

    - The content must be exactly the same as the previous generation
    - You can regenerate each piece of content up to 2 times for free

>     - Any changes to the text, voice settings, or other parameters will require a new generation (that is not free)

Reviewer: This seems to cover the same information as the 1st bullet point. Maybe condense the two into:

```suggestion
    - The content must be exactly the same as the previous generation. Any changes to the text, voice settings, or other parameters will require a new, paid generation
```

>     Free regenerations are useful in case there is a slight distortion in the audio output. According to our internal benchmarks, free regenerations will solve roughly half of issues with quality, with remaining issues usually due to poor training data.

Reviewer: ```suggestion
    Free regenerations are useful in case there is a slight distortion in the audio output. According to ElevenLab's internal benchmarks, regenerations will solve roughly half of issues with quality, with remaining issues usually due to poor training data.
```
